synthesizer/program/src/logic/command/get.rs

```diff
@@ -14,20 +14,21 @@
 
 use crate::{
     traits::{FinalizeStoreTrait, RegistersLoad, RegistersStore, StackMatches, StackProgram},
+    CallOperator,
     Opcode,
     Operand,
 };
 use console::{
     network::prelude::*,
-    program::{Identifier, Register, Value},
+    program::{Register, Value},
 };
 
 /// A get command, e.g. `get accounts[r0] into r1;`.
 /// Gets the value stored at `operand` in `mapping` and stores the result in `destination`.
 #[derive(Clone, PartialEq, Eq, Hash)]
 pub struct Get<N: Network> {
-    /// The mapping name.
-    mapping: Identifier<N>,
+    /// The mapping.
+    mapping: CallOperator<N>,
```

Comment:
This will break the serialization of programs on the current network.

Current programs with the `Identifier` mapping will fail to deserialize because `CallOperator::Resource` requires an extra `variant` byte.

Reply:
Fixed with new serialization method.